app/javascript/gobierto_data/webapp/components/sets/SQLEditor.vue

```diff
@@ -157,23 +150,26 @@ export default {
     },
     getData() {
       const endPoint = `${baseUrl}/data`
-      const url = `${endPoint}?sql=${this.queryEditor}`
 
-      if (this.queryEditor.includes('LIMIT')) {
+      this.queryEditorLowerCase = this.queryEditor.toLowerCase()
+      if (this.queryEditorLowerCase.includes('limit')) {
         this.queryEditor = this.queryEditor
```

Comment:
I've already changed this in my PR, but look closely your code. Closer. Even closer. Do you see? (whispering) "they're the same"